cmd/osbuild-dnf-json-tests/main_test.go

```diff
@@ -47,25 +35,30 @@ func tearDownTemporaryRepository(dir string) error {
 	return os.RemoveAll(dir)
 }
 
-func TestFetchChecksum(quiet bool, dir string) {
+func TestFetchChecksum(t *testing.T) {
+	dir, err := setUpTemporaryRepository()
+	defer func(dir string) {
+		err := tearDownTemporaryRepository(dir)
+		if err != nil {
+			t.Errorf("Warning: failed to clean up temporary repository.")
+		}
+	}(dir)
+	if err != nil {
+		t.Errorf("Failed to set up temporary repository: %s", err)
```

Comment:
I would prefer `t.Fatalf` because it doesn't make much sense to continue in the test when the repo is not set up.

Reply:
Good point. Thanks!